src/components/profile/profile.tsx

```diff
@@ -312,21 +379,49 @@ const Profile = () => {
             </Button>
           </Card.Body>
         </Card>
-        <Card>
+        <Card className="mb-3">
           <Card.Header>Dataset access</Card.Header>
           <Card.Body>
-            <div>
+            <p>
               {numDatasets ? (
                 <NavLink to="/work-package">
                   You have access to
                   {numDatasets === 1
                     ? " one dataset"
                     : ` ${numDatasets} datasets`}
-                  .
+                  .<br />
+                  Click here to set up your download tokens.
```

Comment:
Can you make this a button? Should look similar to "New contact address". We don't need the link then.

Maybe even list the datasets here, similar to the "Pending access requests" section.

Reply:
Also note that there is a nested <p> tag.